source/preview/partials/footer.hbs

```diff
@@ -0,0 +1,41 @@
+<footer class="sg__footer" role="contentinfo">
```

Comment:
Unsure about this, I'd rather remove it since I rather see this repo as a boilerplate.